### PR TITLE
V12 docs

### PR DESCRIPTION
**Pull request by fhammerschmidt**
There is now an .env file with a pointer to the current **latest** and **next** versions. Renamed the "latest" folder to "v11.0.0". When v12 is released we can just move the latest pointer to it.

### CODE REVIEW

**Comment by fhammerschmidt** [comment]
Works fine now with the redirect! Href tests pass as well. Thanks a lot for your suggestions @cometkim 
I'd appreciate a review.

File: package.json [hunk 1]
```diff
@@ -67,6 +67,7 @@
     "@mdx-js/react": "^2.3.0",
     "autoprefixer": "^10.4.14",
     "cssnano": "^6.0.1",
+    "dotenv": "^16.4.7",
```
Comment (aspeddro) [inline package.json:70]: Is this dependency necessary?

Comment (fhammerschmidt) [reply package.json:70]: Probably not if I just put the version aliases in a JS or JSON file from which all of nextjs config, code and tests can read. Shall I?

Comment (aspeddro) [reply package.json:70]: https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#loading-environment-variables

Comment (fhammerschmidt) [reply package.json:70]: I know but I use the .env in a test script as well.

**Review by aspeddro — APPROVED** [review]

**Review by zth — APPROVED** [review]

**Pull request merged**